Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -3143,7 +3143,7 @@ - (void)_presentLockdownMode
             if (!appDisplayName)
                 appDisplayName = [[NSBundle mainBundle] objectForInfoDictionaryKey:(__bridge NSString *)kCFBundleNameKey];
 
-            UIAlertController *alert = [UIAlertController alertControllerWithTitle:[NSString stringWithFormat:WEB_UI_NSSTRING(@"Lockdown Mode is Turned On For “%@“", "Lockdown Mode alert title"), appDisplayName] message:message.get() preferredStyle:UIAlertControllerStyleAlert];
+            UIAlertController *alert = WebKit::createUIAlertController([NSString stringWithFormat:WEB_UI_NSSTRING(@"Lockdown Mode is Turned On For “%@“", "Lockdown Mode alert title"), appDisplayName], message.get());
```

Comment:
If something is called create(), it should return s RetainPtr. Otherwise it is unclear if we need to release the returned value or not.

Reply:
Will do